### PR TITLE
Remove all Test data files

### LINKED ISSUE #10: test data
Great to have the extensive testing, but 200MB of data is quite a lot for a Julia package. Perhaps it'd be good to put the data in a separate repository (`ArchGDALData`), that is in the `test/REQUIRE` only?

The bulk comes from 2 83MB rasters.
`test/ospy/data4/aster.img`
`test/ospy/data5/aster.img`

### PR DESCRIPTION
**Pull request by yeesian**
closes #10

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+    "pyrasterio/example3.tif",
+    "pyrasterio/float_nan.tif",
+    "pyrasterio/float.tif",
+    "pyrasterio/RGB.byte.tif",
+    "pyrasterio/shade.tif"
+]
+
+for f in remotefiles
+    # create the directories if they don't exist
+    filepath = split(f,"/")
```
Comment (evetion) [inline test/runtests.jl]: It is easier to use `dirname` on the filepath. Also `mkpath` creates multiple directories at once.

File: test/ospy/data2/ut_counties.prj [hunk 2]
```diff
@@ -1 +0,0 @@
-GEOGCS["GCS_North_American_1983",DATUM["D_North_American_1983",SPHEROID["GRS_1980",6378137,298.257222101]],PRIMEM["Greenwich",0],UNIT["Degree",0.0174532925199433]]
```
Comment (evetion) [inline test/ospy/data2/ut_counties.prj]: No blank line at the end! It's good that we delete this. 😉

Comment (yeesian) [reply test/ospy/data2/ut_counties.prj]: is it in the spec to have a blank line at the end? i didn't know that

File: test/runtests.jl (hunk 1 above)
Comment (yeesian) [reply test/runtests.jl]: f026144

File: test/ospy/data2/ut_counties.prj (hunk 2 above)
Comment (evetion) [reply test/ospy/data2/ut_counties.prj]: Not in the prj spec. It' just that many old tools assume textfiles end with a newline. That's why Github actively a symbol for it.

**Pull request merged**